src/tools/spark-sstfile-generator/src/main/scala/com/vesoft/nebula/tools/generator/v2/SparkClientGenerator.scala

```diff
@@ -310,7 +310,7 @@ object SparkClientGenerator {
               val values = (for {
                 property <- valueProperties if property.trim.length != 0
               } yield extraValue(row, property)).mkString(",")
-              (row.getString(vertexIndex), values)
+              (String.valueOf(extraValue(row, vertex)), values)
```

Comment:
If you want to transfer extraValue(row, vertex) into String `extraValue(row, vertex).toString` is better?

Reply:
String.valueOf is used to ensure null transferrd to 'null' as String instead of raising NullPointerException.
Actually the function extraValue has had the type transferred to String. No need is String.valueOf or toString to be used.

Reply:
Actually `extraValue` is using row.getString by the index of fields. But it's more safe. IMO it's a good idea